pkg/schema/objects.go

```diff
@@ -29,6 +31,64 @@ type (
 	}
 )
 
+func TypeFromString(input string) Object {
```

Comment:
Can't we just use schema.Parse() instead of this function? That way it would also cover arrays and the like.